web-api/src/app.ts

```diff
@@ -879,6 +880,7 @@ app.delete(
     lambdaWrapper(getDocumentQCInboxForSectionLambda),
   );
   app.get('/sections/:section/judge', lambdaWrapper(getJudgeInSectionLambda));
+  app.get('/sections/chambers', lambdaWrapper(getJudgesChambersLambda));
```

Comment:
We _could_ call `/sections/judge/users`, a route a few lines up, and use that to construct the chambers data on the frontend, which would prevent us from needing another interactor etc. However, this would also mean that the frontend would be coupled to the particular way in which we store data (i.e., that we fetch user data to build chambers data). I have therefore preferred to create a new endpoint and construct the chambers data on the backend so that the frontend can be agnostic about the details.